src/test/resources/annotation-plugin/pom-generate-report.xml

```diff
@@ -13,14 +13,13 @@
             <plugin>
                 <groupId>uk.gov.justice.maven</groupId>
                 <artifactId>annotation-validator-maven-plugin</artifactId>
-                <version>1.6.6-SNAPSHOT</version>
+                <version>1.0.0-SNAPSHOT</version>
```

Comment:
it's a bit difficult to see from just github. But should these 1.0.0-SNAPSHOT all be ${project.version} ? or a property or left as they are?

Just wondering if they will auto version with a maven release.